config.default.yaml

```diff
@@ -3,12 +3,26 @@
 # SPDX-License-Identifier: CC0-1.0
 
 version: 1.0.0
+
```

Comment:
Is this additional string added with some purpose? 

My feeling is that `tutorial` flag and version are close enough by their meaning. Apart of that, it would be nice to keep the config as compact as we can

Reply:
Hi @ekatef, it was introduced for consistency, as every "section" of the config file is separated by a space. I agree it is pedantic in this case, so I will revert it back.